integrationExamples/mass/index.html

```diff
@@ -0,0 +1,107 @@
+<html>
+  <head>
+      <script>
+        window.massConfig = {
+          inskin: {
+            default: {
+              plr_PubfileId: '000000/mass'
+            }
+          }
+        };
+
+        var PREBID_TIMEOUT = 3300;
+  
+        var adUnits = [{
+          code: 'div-gpt-ad-1460505748561-0',
+          mediaTypes: {
+            banner: {
+              sizes: [[300, 250], [300, 600]],
+            }
+          },
+          bids: [{
+            bidder: 'ix',
+            params: {
+              siteId: '123456',
+              size: [300, 250]
+            }
+          }]
```

Comment:
Are you getting any bids with these parameters?  
On my local I cannot get any bid from `ix`.  
I tried changing my location with VPN, still no luck.

Reply:
I just checked with Index Exchange and publishers are supposed to use a tool provided as part of a toolkit that is proprietary to IX. It allows the publisher to check their MASS integration. (Publishers are not expected to entirely self integrate)

If you would like to test it yourself, I can arrange for you to receive the bid simulation tool which is not intended for a general audience.

*We should probably add a comment to this regard in the integration file.